mysql-test/suite/innodb/t/alter_kill.test

```diff
@@ -141,7 +141,7 @@ XA PREPARE 'x';
 
 -- echo # Attempt to start without an *.ibd file.
 let SEARCH_FILE= $MYSQLTEST_VARDIR/log/my_restart.err;
--- error 1,42
```

Comment:
Please check 5.7 HEAD, where it is "-- error 3"

Reply:
This is how it is in Oracle MySQL: 5.7 has `--error 3`, whereas 8.0 has `--error 1`

Reply:
The same `sync_latch_meta_init()` memory leak.